.ci/Jenkinsfile

```diff
@@ -15,7 +15,7 @@
 // specific language governing permissions and limitations
 // under the License.
 
-@Library('apm@master') _
+@Library('apm@test/github-check') _
```

Comment:
Is this simply for the testing of this PR or is the intention to merge this change into master itself?

Reply:
Yes, I'll revert it